Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+
+members = [
+    "base_layer/blockchain",
+    "base_layer/core",
```

Comment:
I would have put all the `base_layer` folders in the root, and just had the `digital_assets_layer` as `digital_assets`

Reply:
I've been back and forth on this myself, but ultimately came to the conclusion that the base and DA layers are siblings and should sit at the same level.

There is scope for a "common" crate that holds stuff that everyone will use, like error handling, logging, etc.